packages/composite-checkout/test/components/checkout-step.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { shallow } from '../enzyme-adapter';
+
+/**
+ * Internal dependencies
+ */
+import CheckoutStep from '../../src/components/checkout-step';
+
+describe( 'CheckoutStep', function() {
+	const componentWithDefaults = (
+		<CheckoutStep
+			title={ 'Step' }
+			stepNumber={ 1 }
+			isActive={ false }
+			isComplete={ false }
+			stepContent={ <span>empty</span> }
+		/>
+	);
+	it( "Default component is selectable by '.checkout-step'", function() {
```

Comment:
Maybe change this to `it ( 'displays a checkout step' )` and the next one would be `it( 'does not display an active step' )` and then `it( 'does not display a complete step' )`. That way the test strings describe what is happening and the class names are just an implementation detail of the test (and can be changed).